tests/sklearn/test_sklearn_model_export.py

```diff
@@ -5,6 +5,7 @@
 import pickle
 import pytest
 import yaml
+import json
```

Comment:
We previously forgot to import the json module, which is required by the integration test behind the `release` marker: `test_sagemaker_docker_model_scoring_with_default_conda_env`.